pkg/operator/ceph/cluster/crash/reconcile.go

```diff
@@ -117,6 +125,12 @@ func (r *ReconcileNode) reconcile(request reconcile.Request) (reconcile.Result,
 			logger.Errorf("more than one CephCluster found in the namespace %q, choosing the first one %q", namespace, cephCluster.GetName())
 		}
 
+		cephVersion, err := getImageVersion(cephCluster.Spec.CephVersion.Image)
+		if err != nil && !getVersionFailed {
+			logger.Warningf("ceph version note found for cluster %q: %v", cephCluster.Name, err)
```

Comment:
typo: not

Reply:
I'll change the name of this variable: it actually reflects whether or not the get version check has already gone through a full recheck cycle and failed. This bool is intended to prevent warnings showing up on every single reconcile loop until the operator restarts if the version check fails. The loop will continue to look for a version, and if it finds one it will use it, but we'll only write to the logs once about it rather than spamming.

Reply:
Actually I was just referring to "note" --> "not". :)